src/casp/services/README.rst

```diff
@@ -0,0 +1,54 @@
+Cellarium Cloud Services
+========================
+
+This section provides an overview of the various services within the Cellarium Cloud platform.
+
+
+Below is the list of individual services:
+
+.. toctree::
+   :maxdepth: 1
+
+   api_service
+   model_inference_service
+   admin_service
+   db_service
+
+General Description
+-------------------
+
+
+It is required to have a ``src/casp/services/.env`` with ``GOOGLE_SERVICE_ACCOUNT_CREDENTIALS`` variable which includes service account json credentials dumped as a one-line string to use any services.
+There are 2 CAS service types:
+
+* Deployable solution: could be used in Cloud Run or deployed anywhere else
+* Cromwell executable task
+
+Both types are containerized solutions. There are 2 types of docker containers for CAS services:
+
+#. Pytorch powered container
+#. Pytorch + cuda powered container (needs a GPU to be executed)
+
+
+Building Docker Images
+......................
+
+
+.. code-block:: bash
+
+    IMAGE_NAME=us-east4-docker.pkg.dev/dsp-cell-annotation-service/casp-pca/casp_pca_service:1.0-gpu
```

Comment:
I don't think it's likely to result in issues because obviously the user would require the correct permissions to be able to push to this address, but should we move references to our specific gcloud project stuff out of the public docs into maybe a doc that just the team has access to?  Also, is this image name up to date?

Reply:
Would removing the image name and substituting it with a placeholder be enough?

Reply:
I mean enough for this publicly available documentation.

Reply:
I think that makes sense as a solution